gdbinit.py

```diff
@@ -19,39 +19,55 @@
 virtual_env = environ.get("VIRTUAL_ENV")
 
 if virtual_env:
-    no_venv_warning = environ.get("PWNDBG_NO_VENV_WARNING", 0)
-    if not no_venv_warning and not sys.executable.startswith(virtual_env):
-        print("[!] Pwndbg Python virtualenv warning [!]")
-        print(
-            "Found Python virtual environment (VIRTUAL_ENV='%s') while GDB is built with a different Python binary (%s)"
-            % (virtual_env, sys.executable)
-        )
-        print("Assuming that you installed Pwndbg dependencies into the virtual environment")
-        print("If this is not true, this may cause import errors or other issues in Pwndbg")
-        print(
-            "If all works for you, you can suppress this warning by setting PWNDBG_NO_VENV_WARNING=1"
-        )
-        print("")
+    venv_warning = int(environ.get("PWNDBG_VENV_WARNING", 1))
+    py_exe_matches = sys.executable.startswith(virtual_env)
+
+    if not py_exe_matches:
+        if venv_warning:
+            print("WARNING: Pwndbg/GDB run in virtualenv with which it may not work correctly ***")
+            print("  Detected Python virtual environment: VIRTUAL_ENV='%s'" % virtual_env)
+            print("  while GDB is built with different Python binary: %s" % sys.executable)
+            print("  Assuming that you installed Pwndbg dependencies into the virtual environment")
```

Comment:
We may want to make this line more clear and say we'll be adding the venvs site packages to `sys.path`, regardless of whether they hide the warning or not.

Reply:
Tbh I find it annoying to get additional redundant output when I run `gdb`. My `gdb` won't start with Pwndbg if I am in wrong virtualenv anyway as other venvs doesn't have pwndbg deps installed :D.

Reply:
Oh, my original comment might have been unclear. I'm not suggesting to show always show some warning, I'm saying that in this original message, we don't mention anywhere that we're going to be modifying `sys.path`, we just say the venv might cause problems. I feel like we should mention that.

Reply:
But I see now that we let the user know this later on, so you don't need to do this.